tests/test_file.py

```diff
@@ -9,6 +9,7 @@
 
 from pages.page_object import MySiteHomePage
 from unittestzero import Assert
+from urlparse import urljoin
```

Comment:
Nit: Standard library imports should be at the top. See http://legacy.python.org/dev/peps/pep-0008/#imports